### PR TITLE
added images

### PR DESCRIPTION
**Pull request by sharlak1**
Changes:
Changed footer image and alignment.
Header image was present in the correct format.
Other images ae yet to be received.

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { BULogo } from '../assets/assets.index';
 export const MainFooter: React.FC = () => {
   return (
     <div style={{ width: '100vw%' }}>
-      <img src={BULogo} style={{ display: 'block', marginLeft: 'auto', padding: '20px' }} />
+      <img src={BULogo} style={{ display: 'flex', alignItems: 'left', padding: '20px', width: '648px' }} />
```
Comment (IanSaucy) [inline src/components/Footer.tsx:7]: Can you switch this from inline style to using the corresponding `.less` file? In addition, switch from `px` to something responsive. `rem`, `em` would probably be a good choice. `vh,vw` could also work well.

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Just updated the styles and it should be good.

**Comment by IanSaucy** [comment]
Deployed URL: https://8162187b.uncounteddeath.pages.dev/

File: src/components/Footer.tsx (hunk 1 above)
Comment (sharlak1) [reply src/components/Footer.tsx:7]: Sure Ian,  I'll make these changes and commit them.

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Just one more dangling pixel value (:

**Review by IanSaucy — APPROVED** [review]
lgtm

**Pull request merged**